tests/unit/utils/test_formatter.py

```
@@ -0,0 +1,88 @@
+"""Tests for the formatter module."""
+
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)